packages/plugins/graphql/package.json

```diff
@@ -50,8 +50,8 @@
   "dependencies": {
     "@apollo/server": "4.10.0",
     "@as-integrations/koa": "1.1.1",
-    "@graphql-tools/schema": "8.5.1",
-    "@graphql-tools/utils": "^8.13.1",
+    "@graphql-tools/schema": "10.0.3",
+    "@graphql-tools/utils": "^10.1.3",
```

Comment:
More general question: how to you pick between using a pinned version or a range?